src/main/java/persistence/sql/dml/QueryBuilder.java

```diff
@@ -1,38 +1,35 @@
 package persistence.sql.dml;
 
-import persistence.sql.dml.keygenerator.KeyGenerator;
-
-import java.lang.reflect.Field;
+import persistence.sql.dialect.Dialect;
 
 public class QueryBuilder {
-    private EntityTableMeta entityTableMeta;
-    private EntityColumns entityColumns;
-
-    public QueryBuilder(Object object) {
-        this.entityTableMeta = EntityTableMeta.of(object.getClass());
-        this.entityColumns = EntityColumns.of(object);
+    private final EntityTableMeta entityTableMeta;
+    private final EntityColumns entityColumns;
+    private final Dialect dialect;
+
+    public QueryBuilder(Class<?> clazz, final Dialect dialect) {
+        this.entityTableMeta = EntityTableMeta.of(clazz);
+        this.entityColumns = EntityColumns.of(clazz);
+        this.dialect = dialect;
     }
 
-    public String createInsertQuery(final KeyGenerator keyGenerator) {
+    public String createInsertQuery(Object object) {
         return String.format("insert into %s (%s) values (%s)", this.entityTableMeta.name(), this.entityColumns.names(),
-                entityColumns.insertValues(keyGenerator));
+                entityColumns.insertValues(object, dialect));
     }
 
     public String createFindAllQuery() {
         return String.format("select %s from %s", this.entityColumns.names(), this.entityTableMeta.name());
     }
 
-    public String createFindByIdQuery() {
-        return String.format("%s where %s = %s", createFindAllQuery(),
-                this.entityColumns.primaryField().getName(),
-                entityColumns.values());
+    public String createFindByIdQuery(Long id) {
+        return String.format("%s where %s = %dL", createFindAllQuery(),
+                this.entityColumns.primaryFieldName(), id);
     }
 
-    public String createDeleteQuery() {
-        final Field primaryField = this.entityColumns.primaryField();
-
+    public String createDeleteQuery(Object object) {
         return String.format("delete from %s where %s = %s", this.entityTableMeta.name(),
```

Comment:
뭔가 Dialect 냄새가 나는군요 😄 
```suggestion
        return String.format(dialect.blabla, this.entityTableMeta.name(),
```